scripts/install-python.ps1

```diff
@@ -81,4 +80,5 @@ if ($PythonExecutable.count -eq 0) {
 } else {
     Write-Host "Python 3.7.3 is already installed." -ForegroundColor Green
 }
-Install-Virtualenv
+
+Install-Virtualenv -ErrorAction Stop
```

Comment:
Error action stop causes all throws to stop the script (assuming it not caught.)